src/ServiceControl.Config/UI/Shell/ShellView.xaml.cs

```diff
@@ -11,7 +11,11 @@ public ShellView()
         {
             InitializeComponent();
 
-            Activated += (s, e) => IoC.Get<IEventAggregator>().PublishOnUIThread(new RefreshInstances());
+            Activated += (s, e) =>
+            {
+                //IoC.Get<IEventAggregator>().PublishOnUIThread(new RefreshInstances());
```

Comment:
Why no longer over the event broker?

Reply:
Because it shortcuts a check that's done in the Shell. If you have no active view (no instance installed) and install a new license this code will blow up with a NRE. 

The code that goes through the shell (below) has this check and is the right way to do it anyway (making a call to its ViewModel rather than an event being raised directly from the View).